integrations/agora-sdk-plugin.md

```diff
@@ -97,25 +97,21 @@ After activating the Symbl Conversation Intelligence Extension, click the **View
 
 This section walks you through the steps necessary to set up the Symbl Conversation Intelligence extension in your mobile application.
 
-1. Download the [Symbl Extension](https://cdn-agora.symbl.ai/agora-symblai-filter-debug.aar) (if you haven't already).
- 
-2. Add the `.aar` file as a dependency to your application. 
-![agora-creds](/img/agora-arr-files.png)
-
-3. Add the following information into your `build.gradle` module file:
+1. Add the following information into your `build.gradle` module file:
 
 ```js
 implementation fileTree(include: ['*.jar'], dir: 'libs')
```

Comment:
Do we still need this if we are not importing files from the filesystem anymore?

Reply:
This will import the Agora SDK Jar files and libs folder (used for android hardware).
Not specific to Symbl.

Should I remove this ?

Reply:
Don't think there are specific steps to import files from the filesystem according to https://docs.agora.io/en/Video/start_call_android?platform=Android. Is there any other step required here?

Reply:
We can skip the first dependency as this is specific to load the agora jar files from file system.

In their newer upgrades, Agora will also push the dependency to maven central for Video SDK.
In that case, there will be no need to change our documentation.

Removing this dependency in a new commit.